src/Sentry.NLog/LogLevelExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+
+using NLog;
+
+using Sentry.Protocol;
+
+namespace Sentry.NLog
+{
+    internal static class LogLevelExtensions
+    {
+        private static readonly IDictionary<LogLevel, SentryLevel> SentryLevelMap = new Dictionary<LogLevel, SentryLevel>
+        {
+            { LogLevel.Debug, SentryLevel.Debug },
+            { LogLevel.Error, SentryLevel.Error },
+            { LogLevel.Fatal, SentryLevel.Fatal },
+            { LogLevel.Info,  SentryLevel.Info },
+            { LogLevel.Trace, SentryLevel.Debug },
+            { LogLevel.Warn,  SentryLevel.Warning },
+        };
+
+        public static SentryLevel? ToSentryLevel(this LogLevel loggingLevel)
+        {
+            if (SentryLevelMap.TryGetValue(loggingLevel, out SentryLevel level))
+                return level;
+
+            return null;
+        }
+
+        private static readonly IDictionary<LogLevel, BreadcrumbLevel> BreadcrumbLevelMap = new Dictionary<LogLevel, BreadcrumbLevel>
+        {
+            { LogLevel.Debug, BreadcrumbLevel.Debug },
+            { LogLevel.Error, BreadcrumbLevel.Error },
+            { LogLevel.Fatal, BreadcrumbLevel.Critical },
+            { LogLevel.Info, BreadcrumbLevel.Info },
+            { LogLevel.Trace, BreadcrumbLevel.Debug },
+            { LogLevel.Warn, BreadcrumbLevel.Warning },
+        };
+
+        public static BreadcrumbLevel ToBreadcrumbLevel(this LogLevel level)
+        {
+            return BreadcrumbLevelMap[level];
```

Comment:
A `LogLevel` out of range of the enum would throw an exception here. If a map lookup fails you could return `(BreadcrumbLevel)level` to pass along the out of range value or simply pick `Info` which is default.

Out of curiosity: Why the choice of a map here instead of plane `switch/case`?
One benefit I see for the `switch` is that it takes no memory to hold the static mapping instances.
Another reason to use `switch` is to be consistent with the other logging integrations.

Reply:
I basically did that for brevity's sake, but I hadn't considered the memory need. I'll change it to a switch.